src/components/ActivityIndicator/index.tsx

```diff
@@ -52,11 +29,20 @@ type ActivityIndicatorProps = {
    * Label should be defined for accessibility, to indicate what is loading
    */
   label?: string
-  id?: string
 }
 
-const ActivityIndicator: FunctionComponent<ActivityIndicatorProps> = ({
-  id: idProp,
+const Text = styled('text', {
```

Comment:
Does the `text` tag exists? I couldn't find it on MDN, a `span` isn't a better option?

Reply:
This is a svg specific tag

https://developer.mozilla.org/en-US/docs/Web/SVG/Element/text